infra/ansible/roles/fetch_srcs/defaults/main.yaml

```diff
@@ -1,3 +1,3 @@
 # See https://docs.ansible.com/ansible/latest/collections/ansible/builtin/git_module.html#parameter-version
-pytorch_git_rev: HEAD
+pytorch_git_rev: v2.1.0-rc2
```

Comment:
Hmm, this doesn't look right. The git rev should be set by the build trigger config.